main.go

```diff
@@ -83,6 +88,8 @@ var (
 	healthProbeAddr                string
 	enableLeaderElection           bool
 	maxHttpRequestBodySize         int64
+	observabilityServiceEndpoint   string
+	observabilityServiveSeed       string
```

Comment:
```suggestion
	observabilityServiceSeed       string
```

Reply:
I think this is not being used. I suggested it before as a possible replacement for a fixed attributes that was being set to the resource: https://github.com/Kuadrant/authorino/pull/380#discussion_r1112958882

Reply:
Yes, I'll remove this